slackgpt/main.py

```diff
@@ -64,7 +66,8 @@ def message(payload: dict):
 
 if __name__ == "__main__":
     lg.info("Starting GPT Thread from main.py")
-    gpt_thread = DavinciApi(handler, args.browser, args.prefix, args.headless)
+    gpt_thread = ChatGPTAPI(handler, args.browser, args.prefix, args.headless) if args.model == "chatgpt" \
```

Comment:
I would suggest you select the class via a dictionary, like this:
```
# somewhere in the beginning:
models = {
  "chatgpt": ChatGPTAPI,
  "davinci": DavinciApi
}

# then when initiating:
message_worker_thread = (handler, args.browser, args.prefix, args.headless, args.model)

# or more verbose:
default_model_class = ChatGPTAPI
model_class = models.get(args.model, default_model_class )
message_worker_thread  = model_class (handler, args.browser, args.prefix, args.headless, args.model)
```

Reply:
That way you dont need the if there, makes it easier to read IMO

Reply:
or you could also use an extra function that does that for you, might be a better idea tbh